examples/first.cpp

```diff
@@ -40,13 +40,13 @@ loops::Func gencode(loops::Context& ctx)
 int main(int argc, char** argv)
 {
     {//Function usage
-        loops::Context ctx(loops::Backend::makeAarch64Compiler());
+        loops::Context ctx;
         loops::Func lfunc = gencode(ctx);
-        std::cout<<"======--BYTECODE-LISTING--======="<<std::endl;
+        std::cout << "======--BYTECODE-LISTING--=======" << std::endl;
         lfunc.printBytecode(std::cout);
-        std::cout<<"======--AARCH64--LISTING--======="<<std::endl;
+        std::cout << "======--AARCH64--LISTING--=======" << std::endl;
```

Comment:
as far as I understand, the sample should compile and run on any architecture. Please, retrieve a readable architecture name ("AARCH64", "INTEL64", etc.) and put it instead of literal "AARCH64"

Reply:
Done.